components/definition/definition.tsx

```diff
@@ -0,0 +1,24 @@
+import ReactMarkdown from 'react-markdown/with-html'
+
+export type DefinitionProps = {
+  readonly slug: string
+  readonly meta: DefinitionMeta
+  readonly content: string
+}
+
+type DefinitionMeta = {
+  readonly question: string
+}
+
+export default function Definition({slug, meta: { question }, content}: DefinitionProps) {
```

Comment:
Try to refactor this one in the nearest PR to send proper model into this dumb component